src/homepage/Running_SoS.ipynb

```diff
@@ -561,6 +561,20 @@
     "although there have been several issues with certain versions of conda (e.g. ([libgcc_s_seh-1.dll under windows](https://github.com/ContinuumIO/anaconda-issues/issues/777) and [missing libreadline under mac](https://github.com/ContinuumIO/anaconda-issues/issues/6312))."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <a id=\"Ruby\"></a><img src=\"img/Ruby.png\" style=\"width:32pt;height:32pt;margin-right:15pt;\">Ruby"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use Ruby with SoS, you will need to install [iRuby](https://github.com/SciRuby/iruby) by following [iRuby Kernel homepage](https://github.com/SciRuby/iruby). After that, install [daru](https://github.com/SciRuby/daru) and [NMatrix](https://github.com/SciRuby/nmatrix)."
```

Comment:
Install `daru` ... with command `gem install daru nmatrix`? This is standard command so I suppose we will not get into trouble for including them.

Reply:
Yup! Unlike kernels, including the instruction of installing library is OK...